tracer/src/Datadog.Trace/Debugger/Snapshots/Redaction.cs

```diff
@@ -0,0 +1,292 @@
+// <copyright file="Redaction.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security;
+using System.Text;
+using Datadog.Trace.Debugger.Helpers;
+
+namespace Datadog.Trace.Debugger.Snapshots
+{
+    internal static class Redaction
+    {
+        private static readonly Type[] AllowedCollectionTypes =
```

Comment:
This has nothing to do with redaction, so seems inappropriate that it's in this class?

Reply:
This comment is relevant to everything in this file up to line 61

Reply:
Will separate these concerns in a later PR.
Resolving.